lars-reimann/api-editor
# feat: Persist Python Packe and Annotation Store to Local Storage.

Closes #91.

## Summary of Changes
Persist imported JSON-Files to Local Storage and synchronize changes of Annotation Store with it.

## Screenshots (if necessary)
There are no visible changes.

## Testing instructions
1. Make changes to annotations, close your window or browser and reopen the application.
   Your changes should be there again.
2. Import a new different package. Then again close your window or browser and reopen the application.
   The application should not load the old JSON.
